nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/provenance/provenance-content.jsp

```diff
@@ -17,8 +17,8 @@
 <%@ page contentType="text/html" pageEncoding="UTF-8" session="false" %>
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 <div id="provenance">
-    <span id="intial-component-query" class="hidden"><c:out value="${param.componentId}"/></span>
```

Comment:
Thanks for spotting this. Couldn't find a CSS but found a javascript file referencing to it. Added to a second commit.